go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/openshift-kni/numaresources-operator
 
-go 1.22.0
-
-toolchain go1.22.8
+go 1.23.0
```

Comment:
bumping deps is fine. Bumping the golang version is fine (but we need to check the builder images as well). Bumping them both in the same commit is not fine

Reply:
I saw it already bumped on separate PR